k8s/helm-charts/seldon-core-v2-setup/templates/seldon-v2-components.yaml

```diff
@@ -1663,6 +1663,8 @@ spec:
       - containerPort: 9000
         name: server-http
         protocol: TCP
+      - containerPort: 8082
```

Comment:
@michaelcheah did you check that mlserver indeed expose metrics under this port?

Reply:
Ok, it does.

Reply:
Ok, it does.